flink-core/src/test/java/org/apache/flink/api/common/state/StateTtlConfigTest.java

```diff
@@ -71,4 +74,22 @@ public void testStateTtlConfigBuildWithCleanupInBackground() {
         assertThat(incrementalCleanupStrategy.runCleanupForEveryRecord(), is(false));
         assertThat(rocksdbCleanupStrategy.getQueryTimeAfterNumEntries(), is(1000L));
     }
+
+    @Test
+    public void testStateTtlConfigBuildWithNonPositiveCleanupIncrementalSize() {
+        ArrayList<Integer> illegalCleanUpSizes = new ArrayList<Integer>();
+        illegalCleanUpSizes.add(0);
+        illegalCleanUpSizes.add(-2);
```

Comment:
Generally, I prefer to use `List<Integer> illegalCleanUpSizes = Arrays.asList(0, -2);` to create the candidate list.

Reply:
> Generally, I prefer to use `List<Integer> illegalCleanUpSizes = Arrays.asList(0, -2);` to create the candidate list.

Hi Myasuka , done.